imports/ofx/account.go

```diff
@@ -0,0 +1,58 @@
+package ofx
+
+import (
+	"github.com/aclindsa/xml"
+	"gorm.io/gorm"
+)
+
+type Account struct {
+	gorm.Model
+	Currency     string
+	BankID       string
+	AccountID    string
+	Type         string
+	Transactions []Transaction
+}
+
+func (Account) TableName() string {
+	return tablePrefix + "accounts"
+}
+
+func (a *Account) UnmarshalXML(decoder *xml.Decoder, start xml.StartElement) error {
+	type Alias Account
+
+	var data struct {
+		Alias
+		AccountInfo struct {
+			Currency string `xml:"CURDEF"`
+			Meta     struct {
+				BankID    string `xml:"BANKID"`
+				AccountID string `xml:"ACCTID"`
+				Type      string `xml:"ACCTTYPE"`
+			} `xml:"BANKACCTFROM"`
+			TransactionsStruct struct {
```

Comment:
`TransactionsStruct` name seems too code-related for me 😬  How's about something like this more related to the origin XML?
```suggestion
			BankTransactionsList struct {
```